app/views/users/show_user.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Your Fweets, <%="#{@user.username}"%>!</h1>
+
+<ul>
+  <% @tweets.each do |a| %>
+    <% if a.user_id == @user.id %>
```

Comment:
I would move this check into the controller. It would be more efficient to verify the association through ActiveRecord than repeating the validation for each tweet in this loop.